app/views/validate.py

```diff
@@ -44,14 +49,18 @@ def validate_schema(data):
         return Response(status=400, response="Could not parse JSON")
 
     response = {}
+    try:
+        ajv_response = requests.post(AJV_VALIDATOR_URI, json=json_to_validate)
+        ajv_response_dict = ajv_response.json()
 
-    schema_validator = SchemaValidator(json_to_validate)
-    schema_validator.validate()
+        if ajv_response_dict:
+            response["errors"] = ajv_response_dict["errors"]
+            logger.info("Schema validator returned errors", status=400)
+            return jsonify(response), 400
 
-    if len(schema_validator.errors) > 0:
-        response["errors"] = schema_validator.errors
-        logger.info("Schema validator returned errors", status=400)
-        return jsonify(response), 400
+    except requests.exceptions.ConnectionError:
```

Comment:
Catch all request exceptions?. We don't want to hit the custom python validator if ajv isn't run, which could be other things than connection error.

Reply:
Personally i don't like broad exceptions, i will update it to capture all request exception

Reply:
Ya, I didn't mean just `Exception` just request acception.